### PR TITLE
BZ1946255: removed the title and procedure from the VMware vSphere backup section and made the snapshot limitation paragraph a note

### PR DESCRIPTION
**Pull request by chinmayi-chandrasekar**
Applicable for 4.5+

https://bugzilla.redhat.com/show_bug.cgi?id=1946255

Doc preview link: https://deploy-preview-34356--osdocs.netlify.app/openshift-enterprise/latest/storage/persistent_storage/persistent-storage-vsphere?utm_source=github&utm_campaign=bot_dp

Requires QA approval from @duanwei33 
Requires engineering approval from @jsafrane

### CODE REVIEW

**Comment by mburke5678** [comment]
@chinmayi-chandrasekar LGTM

**Review by lpettyjo — COMMENTED** [review]
I'd reword as:
{product-title} provisions new volumes as independent persistent disks that can freely attach and detach the volume on any node in the cluster. Consequently, you cannot back up volumes that use snapshots, or restore volumes from snapshots. For more information, see link:https://docs.vmware.com/en/VMware-vSphere/6.7/com.vmware.vsphere.vm_admin.doc/GUID-53F65726-A23B-4CF0-A7D5-48E584B88613.html[Snapshot Limitations].

Otherwise, LGTM.

**Comment by lpettyjo** [comment]
I'd also have @jsafrane from engr ack too.
